backend/mora/app.py

```diff
@@ -70,20 +70,28 @@ def handle_invalid_usage(error):
 
         return error.get_response(flask.request.environ)
 
-    if app.env != 'production':
-        @app.route('/')
-        @app.route('/<path:path>')
-        def root(path=''):
-            '''Serve static files.
-
-            Disabled in production since you should configure the
-            proxy to do so.
+    # We serve index.html and favicon.ico here. For the other static files,
+    # Flask automatically adds a static view that takes a path relative to the
+    # `flaskr/static` directory.
+
```

Comment:
I think we should preserve the logic that disables these routes when running in production mode.

Reply:
From the commit message:
> The reason against serving static files via python rather than a proper web
> server is performance. Since we only serve few kilobytes of html, js, and css
> the negative penalty negligible compared to the REST calls.
> 
> The simplicity of having a single docker container serving everything is nice
> compared to making sure the static assets are served by the proxy and kept in
> sync on updates or by having a whole web server in this container to serve just
> a few small files.